### PR TITLE
Added modal functionality to insert comment to audit logs when un/locking account.

### PR DESCRIPTION
**Pull request by abpai94**
#127

### CODE REVIEW

File: docs/audit.rst [hunk 1]
```diff
@@ -25,6 +26,7 @@ Example:
     "done_by":"Mickey",
     "action":"lockaccount",
     "result":"accountlocked"
```
Comment (coudot) [inline docs/audit.rst:28]: A comma is missing here

Comment (coudot) [reply docs/audit.rst:28]:
```suggestion
    "result":"accountlocked",
```

File: htdocs/lockaccount.php [hunk 2]
```diff
@@ -6,6 +6,7 @@
 $result = "";
 $dn = "";
 $password = "";
+$comment = $_POST["comment"];
```
Comment (coudot) [inline htdocs/lockaccount.php:9]: Like other POST params, you should check the presence of the param with isset (see example with "dn" just below)

File: lang/en.inc.php [hunk 3]
```diff
@@ -32,6 +34,7 @@
 $messages['label_authtimestamp'] = "Last authentication";
 $messages['label_businesscategory'] = "Business category";
 $messages['label_carlicense'] = "Car license";
+$messages['label_comment'] = "Comment";
```
Comment (coudot) [inline lang/en.inc.php:37]: label_x is reserved to tranlsation of LDAP attributes, another key like "comment" and rename "comment" key to "insert_comment"

**Review by coudot — CHANGES_REQUESTED** [review]
Please look at suggestions.

And configuration parameters are missing for now.

**Review by coudot — APPROVED** [review]

**Comment by coudot** [comment]
Hello @abpai94 

thanks for the modifications, all is working fine!

Before merging it, I think we should:
* Add missing translation string (I can do it)
* Decide if we want to change the color of the button if a comment is expected (I don't think it is needed)
* I would suggest to change the modal header title to the action (Unlock, Lock, ...) so the admin knows which action is concerned by the comment
* Do not store the comment key in audit if comment is empty
* See if the comment should be mandatory. Maybe we could add another parameter to manage this.

**Comment by abpai94** [comment]
Hello @coudot 

The following three points have been covered:

> - [x] I would suggest to change the modal header title to the action (Unlock, Lock, ...) so the admin knows which action is concerned by the comment 
> - [x] Do not store the comment key in audit if comment is empty 
> - [x] See if the comment should be mandatory. Maybe we could add another parameter to manage this.

**Comment by coudot** [comment]
Regression found:
```
PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function auditlog(), 5 passed in /usr/local/service-desk/htdocs/checkpassword.php on line 44 and exactly 6 expected in /usr/local/service-desk/lib/audit.inc.php:2\nStack trace:\n#0 /usr/local/service-desk/htdocs/checkpassword.php(44): auditlog()\n#1 /usr/local/service-desk/htdocs/index.php(202): require_once('...')\n#2 {main}\n  thrown in /usr/local/service-desk/lib/audit.inc.php on line 2'
```

Will fix it in master

**Pull request merged**